page.tsx

```diff
@@ -88,7 +99,11 @@ export default function createPage(swingsetOptions: SwingsetOptions = {}) {
           </div>
           <Nav navData={filteredNav} />
         </div>
-        <div className={s.stage}>
+        <div className={classNames(s.stage, { [s.isFullscreen]: isFullscreen})}>
+          <button className={s.fullscreenBttn} aria-labelledby="buttonLabel" onClick={handleFullscreenBttnClick}>
+            { isFullscreen ? (<CloseFullscreenIcon />) : (<FullscreenIcon />)}
+            <span id="buttonLabel" hidden>{ isFullscreen ? 'Close fullscreen' : 'Enter fullscreen'}</span>    
+          </button>
```

Comment:
Swapped in a `button` and added hidden text. Verified that this works with VO.

Reply:
I think the hidden attribute is used to mark content as not presented to the user which I would have thought hidden from VO too. Typically will reach for using `@reach/visually-hidden` to provide visually hidden text that is accessible to screen readers.

Reply:
I think this works because the `span` is hidden but it gets referenced by the `button` via the `aria-labelledby` attribute.

Either way, better to follow existing patterns so I went ahead and implemented `@reach/visually-hidden`!